Fix scroll overflow in extension link with fragments

alessandrojean opened the pull request:
Closes #1048.

[Gravação de tela de 2023-10-05 18-32-42.webm](https://github.com/tachiyomiorg/website/assets/14254807/abbf2d96-bcf2-4cdb-bc64-872d29d5b5d9)

Soitora commented:
Are you able to make it so that the accessibility tab thing doesn't focus too?

![image](https://github.com/tachiyomiorg/website/assets/10836780/5816453b-0557-46f7-a517-5380f9c7c377)

Maybe we should also set Z-pos on stuff

alessandrojean commented:
Changed the style to remove the outline. I also made the package name a little small and fixed the bad color contrast when it's the current target by changing the gray color to one from the theme.

| Dark | Light |
| -- | -- |
| ![image](https://github.com/tachiyomiorg/website/assets/14254807/5f68dbb6-d8f9-429b-905d-6480b654748d) | ![image](https://github.com/tachiyomiorg/website/assets/14254807/427d58f2-973d-46ab-ba31-4cb42138c680) |
